roomserver/storage/shared/storage.go

```diff
@@ -439,8 +439,18 @@ func (d *Database) Events(
 }
 
 func (d *Database) events(
-	ctx context.Context, txn *sql.Tx, eventNIDs []types.EventNID,
+	ctx context.Context, txn *sql.Tx, inputEventNIDs types.EventNIDs,
 ) ([]types.Event, error) {
+	sort.Sort(inputEventNIDs)
```

Comment:
Why sort?

Reply:
You know, that's a good question. The sort was there before (albeit in a different place in the function) and I now don't remember why.